alexthompson207/rancid-tomatillos
# Feature/form search

## Is this a fix or a feature?

This is a feature.

## What is the change?

Change the `form` function component to a class component so that it can be a controlled form and hold state. Add conditionals to the `Movies.js` component view to display filtered movies or all movies, and add error handling if searched movies doesn't match any movie. Then add styling to the form.

## What does it fix?

N/a.

## Where should the reviewer start?

In the `Form.js` component file then the `App.js` file.

## How should this be tested?

Pull down this branch, open the site locally. As a user you should be able to type into the search form and matching movie titles should render, if no matches are found a user should be able to clear their search and see the all movies view.
